### PR TITLE
Permissioning overview

### PR DESCRIPTION
**Pull request by MadelineMurray**
Added overview of Permissions functionality

### CODE REVIEW

File: docs/Permissions/Permissioning.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    `nodes-whitelist=["enode://6f8a80d14311c39f35f516fa664deaaaa13e85b2f7493f37f6144d86991ec012937307647bd3b9a82abe2974e1407241d54947bbb39763a4cac9f77166ad92a0@192.168.0.9:4567","enode://6f8a80d14311c39f35f516fa664deaaaa13e85b2f7493f37f6144d86991ec012937307647bd3b9a82abe2974e1407241d54947bbb39763a4cac9f77166ad92a0@192.169.0.9:4568"]`
+
+Node whitelisting is at the node level. That is, each node in the network has a [`permissions_config.toml`](#permissions-configuration-file)
+file in the [data directory](../Reference/Pantheon-CLI-Syntax.md#data-path) for the node.  
+
+To add or remove nodes from the nodes whitelist when the node is running, use the JSON-RPC API methods:
+ 
+* [perm_addNodesToWhitelist](../Reference/JSON-RPC-API-Methods.md#perm__addnodestowhitelist)
+* [perm_removeNodesFromWhitelist](../Reference/JSON-RPC-API-Methods.md#perm_removeNodesFromWhiteList)
+* [perm_getNodesWhitelist](../Reference/JSON-RPC-API-Methods.md#perm_getNodesWhiteList)
```
[the remark this answers is not in the document]
Comment (macfarla) [reply docs/Permissions/Permissioning.md]: or maybe say "To view, add or remove ..."

File: docs/Permissions/Permissioning.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+The bootnodes must be included in the nodes whitelist. 
+
+!!! example 
+    If you start Pantheon with:
+    
+     ```bash
+     --bootnodes="enode://7e4ef30e9ec683f26ad76ffca5b5148fa7a6575f4cfad4eb0f52f9c3d8335f4a9b6f9e66fcc73ef95ed7a2a52784d4f372e7750ac8ae0b544309a5b391a23dd7@127.0.0.1:30303","enode://2feb33b3c6c4a8f77d84a5ce44954e83e5f163e7a65f7f7a7fec499ceb0ddd76a46ef635408c513d64c076470eac86b7f2c8ae4fcd112cb28ce82c0d64ec2c94@127.0.0.1:30304","enode://7b61d5ee4b44335873e6912cb5dd3e3877c860ba21417c9b9ef1f7e500a82213737d4b269046d0669fb2299a234ca03443f25fe5f706b693b3669e5c92478ade@127.0.0.1:30305" 
+     ```
+    
+    The `nodes-whitelist` in `permissioning_config.toml` must contain the specified bootnodes.
```
Comment (macfarla) [inline docs/Permissions/Permissioning.md]: do you want to say what happens if they aren't?

File: docs/Permissions/Permissioning.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (MadelineMurray) [reply docs/Permissions/Permissioning.md]: Changed to 'To view or update'

File: docs/Permissions/Permissioning.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
+description: Pantheon Permissions feature
+<!--- END of page meta data -->
+
+# Permissions 
+
+A permissioned network is a network where only specified nodes and accounts (participants) can participate.
```
[the remark this answers is not in the document]
Comment (MadelineMurray) [reply docs/Permissions/Permissioning.md]: Added a sentence to make this clearer

File: docs/Permissions/Permissioning.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+
+To enable node whitelisting, specify the [`--permissions-nodes-enabled`](../Reference/Pantheon-CLI-Syntax.md#permissions-nodes-enabled) option 
+when starting Pantheon. 
+
+The `PERM` API methods are not enabled by default. Use the [`--rpc-http-api`](../Reference/Pantheon-CLI-Syntax.md#rpc-http-api) 
+or [`--rpc-ws-api`](../Reference/Pantheon-CLI-Syntax.md#rpc-ws-api) options to enable the `PERM` API methods.
+
+## Account Whitelisting 
+
+Accounts whitelisting is specified by the accounts whitelist in the [`permissions_config.toml`](#permissions-configuration-file) file.
```
[the remark this answers is not in the document]
Comment (MadelineMurray) [reply docs/Permissions/Permissioning.md]: Fixed throughout

File: docs/Permissions/Permissioning.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+    
+!!! example "Example Different Account Whitelists"
+    Node 1 Whitelist = [Account A, Account B]
+    
+    Node 2 Whitelist = [Account B, Account C]
+    
+    Mining Node Whitelist = [Account A, Account B]
+    
+    Account A submits a transaction on Node 1. Node 1 validates and propagates the transaction. The Mining Node receives the transaction, 
+    validates it is from an account in it's account whitelist, and includes the transaction in the block. Node 2 receives and adds the block created by
```
[the remark this answers is not in the document]
Comment (MadelineMurray) [reply docs/Permissions/Permissioning.md]: Reworded

File: docs/Permissions/Permissioning.md [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+    Node 2 now has a transaction in the blockchain from Account A which does not belong to the account whitelist for Node 2.   
+
+!!! note
+    Each node has a `permissions_config.toml` file which means nodes in the network can have different accounts whitelists. 
+    This means a transaction can be successfully submitted by Node A with the account in the Node A whitelist but rejected by 
+    Node B to which it is propagated if the account is not in the Node B whitelist. 
+    We recommend each node in the network has the same nodes whitelist.
+        
+    On-chain permissioning is under development. On-chain permissioning will use one onchain 
+    node whitelist.
```
[the remark this answers is not in the document]
Comment (MadelineMurray) [reply docs/Permissions/Permissioning.md]: Fixed

File: docs/Permissions/Permissioning.md (hunk 2 above)
Comment (MadelineMurray) [reply docs/Permissions/Permissioning.md]: Added

**Pull request merged**